transports/webrtc/src/tokio/connection.rs

```diff
@@ -144,6 +151,13 @@ impl Connection {
             })
         }));
     }
+
+    /// Fetches the current bandwidth (for all connections). Counters are reset after each call.
```

Comment:
I know this is not perfect from API design, but it's the trade-off. You can browse the commit history.